src/types/seed_phrase.ts

```diff
@@ -0,0 +1,35 @@
+import { Equatable } from './equatable';
+
+const seedPhraseRegex = /^(\b[a-zA-Z]+\b(\s+(?=\b[a-zA-Z]+\b)|$)){12}$/;
```

Comment:
To-Do for @matibat: review this regexp

Reply:
Okay, I now see the point of that positive look-ahead.
1. It has to be a look-ahead to exclude it from the match, and
2. it has to match the next word or the end of the line.
3. Then because of 1. the space is outside the look-ahead because we want to match it.
The purpose is to avoid the trailing space. It would be kind of: "match all words that are followed by more than one space (included) and another word, or the end of the line"

Now, see this:
```suggestion
const seedPhraseRegex = /^(\b[a-zA-Z]+\b(\s+\b|$)){12}$/;
```
The phrase for this regexp would be: "match all words that are followed by more than one space and a word boundary (both included), or the end of the line"
With that regexp I took advantage of `\b` not being an actual character so we can get rid of the look-ahead

Reply:
We can simplify a little more by adding the case `i`nsentitive flag
```suggestion
const seedPhraseRegex =/^(\b[a-z]+\b(\s+\b|$)){12}$/i;
```
although I'm not sure if the seed phrase accepts capitalized characters

Reply:
@arielmelendez what dya think?